### DISCUSSION TITLE
Passing a content collection to a component

### DISCUSSION THREAD

**Discussion by rimien** [nuejs/nue#396] [Q&A]
Hi, first - I am very glad to see Nue and the ideas behind - keep up the good work!

Quick question: Is it possible, to pass (the name of) a content collection to a component? How can I access the frontmatter properties of the collection pages from within the component?
Thanks in advance!

**Comment by tipiirai** [comment]
Yes it is. You can see an example custom component here, which loops a content collection called "posts":

https://nuejs.org/docs/content-collections.html#custom-layouts
